src/main/java/seedu/address/model/person/DisplayPostalCode.java

```diff
@@ -0,0 +1,55 @@
+package seedu.address.model.person;
```

Comment:
is there really a need for a new DisplayPostalCode class?

Reply:
The rationale for the display postal was to make it clearer since the display will contain lots of numbers in our context.
It was the most balanced way out I could find. 
1: I could change postal code class to require the S prefix but it feels pretty lame for the user to have to type S all the time (unless we use it as though it is a parser prefix but that would be inconsistent)
2: I think I tried having both S and non-S postal codes as the same class but it made validation and testing pretty cray (2 isValidPostalCode() are needed I think? And have to know where to call which one)
3: There was no way to do a simple adding of an "S" to the fxml labels as far as I can tell